fuzzbench/jobs.py

```diff
@@ -11,25 +11,41 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Fake jobs."""
+"""Jobs."""
 
+import os
 import subprocess
 import time
 
+BASE_TAG = 'gcr.io/fuzzbench'
 
-def build_image(name):
-    """Build a Docker image."""
+
+def build_base_images(name: str):
+    """Builds a Docker image."""
+    image_tag = os.path.join(BASE_TAG, name)
+    return subprocess.run(
+        ['docker', 'build', '--tag', image_tag,
+         os.path.join('docker', name)],
+        check=True)
+
+
+def build_image(name: str):
+    """Builds a Docker image."""
     print('Building', name)
-    subprocess.run(['docker', '--version'], check=True)
-    time.sleep(3)
+    if name.startswith('base'):
+        if name in ['base-image', 'base-builder', 'base-runner']:
+            build_base_images(name)
+    else:
+        subprocess.run(['docker', '--version'], check=True)
+        time.sleep(3)
```

Comment:
Can we please merge these two functions? It's fine if for now it only works for base images, we can start add support for fuzzer-benchmark runners once https://github.com/google/fuzzbench/pull/543 lands